README.md

```diff
@@ -1 +1,91 @@
-# das-reservations-jobs
\ No newline at end of file
+# das-reservations-jobs
+
+## Build Status
+
+![Build Status](https://sfa-gov-uk.visualstudio.com/Digital%20Apprenticeship%20Service/_apis/build/status/Manage%20Funding/das-reservations-jobs?branchName=master)
+
+## Requirements
+
+DotNet Core 2.2 and any supported IDE for DEV running. Azure function SDK v2 is also required.
```

Comment:
Just remember that 2.2 is not LTS version so 2.1 or 3.1 would be better

Reply:
Agreed - although all the others are 2.2 🤦‍♂ might be worth upgrading to 3 at some point

Reply:
ok i'll leave that then as a separate task